report/node_test.go

```diff
@@ -146,33 +146,34 @@ func TestMergeNodes(t *testing.T) {
 				}),
 			},
 		},
-		"Counters": {
-			a: report.Nodes{
-				"1": report.MakeNode("1").WithCounters(map[string]int{
-					"a": 13,
-					"b": 57,
-					"c": 89,
-				}),
-			},
-			b: report.Nodes{
-				"1": report.MakeNode("1").WithCounters(map[string]int{
-					"a": 78,
-					"b": 3,
-					"d": 47,
-				}),
-			},
-			want: report.Nodes{
-				"1": report.MakeNode("1").WithCounters(map[string]int{
-					"a": 91,
-					"b": 60,
-					"c": 89,
-					"d": 47,
-				}),
-			},
-		},
 	} {
 		if have := c.a.Merge(c.b); !reflect.DeepEqual(c.want, have) {
 			t.Errorf("%s: %s", name, test.Diff(c.want, have))
 		}
 	}
 }
+
+func TestCounters(t *testing.T) {
```

Comment:
> Counters are not set in the probe, so we don't need any backwards-compatibility in report decoding. Similarly they are not set until after all nodes are merged, so we don't need that logic.
(It was a bug if we did use it - merges are supposed to be idempotent).

Could you please add (some of) this is a comment to the test, just to justify that merges are not suppose to work anymore.

Reply:
They _work_ ; it's just that there is no special logic to merge counters now they are just values.  I'll add a note that we don't check this.

Reply:
done